flake.nix

```diff
@@ -4,11 +4,20 @@
   inputs.nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
   inputs.flake-utils.url = "github:numtide/flake-utils";
 
-  outputs = { self, nixpkgs, flake-utils }:
+  outputs = {
+    self,
+    nixpkgs,
+    flake-utils,
+  }:
     flake-utils.lib.eachDefaultSystem (system: {
-      packages.nix-update = nixpkgs.legacyPackages.${system}.callPackage self {
-        src = self;
+      formatter = nixpkgs.legacyPackages.${system}.alejandra;
```

Comment:
I added the formatter here.

Reply:
My bad, I didn't see that at the first glance. edited the comment